robots/rbkairos.urdf.xacro

```diff
@@ -0,0 +1,134 @@
+
+    <!-- *************** -->
+    <!-- Robots Elements -->
+    <!-- *************** -->
+
+    <!-- Here we create the robot elements using the xacro:macros imported at the beggining of this file -->
+
+    <!-- Summit XL base -->
+    <xacro:rbkairos_base prefix="$(arg prefix)" publish_bf="true" hq="${hq}" laser_support="true" />
+
+    <xacro:omni_wheel prefix="$(arg prefix)front_right" parent="$(arg prefix)base_link" reflect="false" hq="${hq}">
+      <origin xyz="${wheel_offset_x} -${wheel_offset_y} ${wheel_offset_z}" rpy="0 0 0"/>
+    </xacro:omni_wheel>
+
+    <xacro:omni_wheel prefix="$(arg prefix)front_left" parent="$(arg prefix)base_link" reflect="true" hq="${hq}">
+      <origin xyz="${wheel_offset_x} ${wheel_offset_y} ${wheel_offset_z}" rpy="0 0 0"/>
+    </xacro:omni_wheel>
+
+    <xacro:omni_wheel prefix="$(arg prefix)rear_left" parent="$(arg prefix)base_link" reflect="true" hq="${hq}">
+      <origin xyz="-${wheel_offset_x} ${wheel_offset_y} ${wheel_offset_z}" rpy="0 0 0"/>
+    </xacro:omni_wheel>
+
+    <xacro:omni_wheel prefix="$(arg prefix)rear_right" parent="$(arg prefix)base_link" reflect="false" hq="${hq}">
+      <origin xyz="-${wheel_offset_x} -${wheel_offset_y} ${wheel_offset_z}" rpy="0 0 0"/>
+    </xacro:omni_wheel>
+
+
+    <!-- IMU -->
+    <xacro:sensor_vectornav frame_prefix="$(arg prefix)imu_" parent="$(arg prefix)base_link">
+      <origin xyz="${imu_offset_x} ${imu_offset_y} ${imu_offset_z}" rpy="0 0 0"/>
+    </xacro:sensor_vectornav>
+
+    <!-- SENSORS -->
+    <xacro:sensor_sick_microscan3 frame_prefix="$(arg prefix)front_laser_" parent="$(arg prefix)base_link" topic_prefix="front_laser" gpu="${gpu}">
+      <origin xyz="${front_laser_offset_x} ${front_laser_offset_y} ${front_laser_offset_z}" rpy="0 ${-PI} ${3/4*PI}" />
+    </xacro:sensor_sick_microscan3>
+    <xacro:sensor_sick_microscan3 frame_prefix="$(arg prefix)rear_laser_" parent="$(arg prefix)base_link" topic_prefix="rear_laser" gpu="${gpu}">
+      <origin xyz="${rear_laser_offset_x} ${rear_laser_offset_y} ${rear_laser_offset_z}" rpy="0 ${-PI} ${-1/4*PI}" />
+    </xacro:sensor_sick_microscan3>
+
+
+<!--     <xacro:sensor_orbbec_astra prefix="$(arg prefix)front_rgbd_camera" parent="$(arg prefix)base_link" prefix_topic="front_rgbd_camera">
+      <origin xyz="${front_rgbd_camera_offset_x} ${front_rgbd_camera_offset_y} ${front_rgbd_camera_offset_z}" rpy="0 0 0"/>
+    </xacro:sensor_orbbec_astra> -->
+
+    <!--xacro:sensor_zed prefix="$(arg prefix)front_rgbd_camera" parent="$(arg prefix)base_link" prefix_topic="front_rgbd_camera">
+      <origin xyz="${front_rgbd_camera_offset_x} ${front_rgbd_camera_offset_y} ${front_rgbd_camera_offset_z}" rpy="0 0 0"/>
+    </xacro:sensor_zed-->
+
+
+
+  </xacro:macro>
+
+  <!-- Gazebo ros control pluggins -->
+  <xacro:ros_control/>
+  <xacro:if value="$(arg ros_planar_move_plugin)">
+    <xacro:omni_steering prefix="$(arg prefix)" publish_tf="true"/>
+  </xacro:if>
```

Comment:
im not sure if these lines work in ros2, is this tested in gazebo? @cgasconb

Reply:
I've modified the xacro to use the ros2 plugin, it worked during the tests